pdf-aam/index.html

```diff
@@ -198,30 +197,30 @@ <h2>PDF-specific Terms & Definitions</h2>
     </section>
 
     <section id="mapping-pdf-object-to-be-named">
-      <h2>Mapping PDF-object to be named to Accessibility APIs</h2>
+      <h2>Mapping PDF Logical Structure Items to Accessibility APIs</h2>
 
       <h3>General Rules for Exposing PDF Semantics via HTML or WAI-ARIA Semantics</h3>
       <p>
-        Unless indicated otherwise, a PDF-object to be named with default Accessible Rich Internet Applications (WAI-ARIA) 1.2 semantics MUST be exposed to the platform accessibility APIs according to
+        Unless indicated otherwise, a PLSI with default Accessible Rich Internet Applications (WAI-ARIA) 1.2 semantics MUST be exposed to the platform accessibility APIs according to
```

Comment:
```suggestion
        Unless indicated otherwise, a <abbr title="PDF Logical Structure Items">PLSI</abbr> with default Accessible Rich Internet Applications (WAI-ARIA) 1.2 semantics MUST be exposed to the platform accessibility APIs according to
```